.eslintrc.js

```diff
@@ -495,25 +503,31 @@ if (process.env.FULL_ESLINT_CHECK) {
       ...config.overrides,
     ],
   };
-} else {
-  config.plugins = [...config.plugins, 'markdown'];
+} else if (process.env.MD_ESLINT_CHECK) {
   config = {
     ...config,
     // Apply the markdown plugin
     plugins: [...config.plugins, 'markdown'],
 
+    parserOptions: {
+      ecmaVersion: 2021,
+      sourceType: 'module',
+      ecmaFeatures: {
+        jsx: true,
+      },
+    },
```

Comment:
Second relevant change. We could revert some of the other changes in this file, let me know.

Reply:
Why do we want a different `parserOptions` (i.e. not `parserOptions.tsconfigRootDir` and `parserOptions.project`) when `MD_ESLINT_CHECK` is true?

Reply:
It's not possible to use type-aware linting rules with any ESLint processors.
Only @typescript-eslint rules that don't require a tsconfig.json are supported.